rca/scholarships/templates/scholarships/index.html

```diff
@@ -15,92 +15,53 @@
-                                        <ul>
-                                            {% for type in required_parent_types %}<li><b>{{ type|title }}</b></li>{% endfor %}
-                                        </ul>
-                                    {% else %}
-                                        <p>{% blocktrans with view.verbose_name_plural as name %}No {{ name }} have been created yet.{% endblocktrans %}
-                                        {% if user_can_create %}
-                                            {% blocktrans with view.create_url as url %}
-                                                Why not <a href="{{ url }}">add one</a>?
-                                            {% endblocktrans %}
-                                        {% endif %}</p>
-                                    {% endif %}
-                                </div>
-                            {% else %}
-                                {% result_list %}
+            {% block header_extra %}
+                <div class="right header-right">
+                    {% if view.list_export or user_can_create %}
+                        <div class="actionbutton">
+                            {% if user_can_create %}
+                                {% include 'modeladmin/includes/button.html' with button=view.button_helper.add_button %}
                             {% endif %}
-                        {% endblock %}
-                    </div>
-
-                    {% block pagination %}
-                        <nav class="pagination {% if view.has_filters and all_count %}col9{% else %}col12{% endif %}" aria-label="{% trans 'Pagination' %}">
-                            <p>{% blocktrans with page_obj.number as current_page and paginator.num_pages as num_pages %}Page {{ current_page }} of {{ num_pages }}.{% endblocktrans %}</p>
-                            {% if paginator.num_pages > 1 %}
-                                <ul>
-                                    {% pagination_link_previous page_obj view %}
-                                    {% pagination_link_next page_obj view %}
-                                </ul>
+                            {% if view.list_export %}
+                                <div class="dropdown dropdown-button match-width col">
+                                    <a href="?export=xlsx&{{ request.GET.urlencode }}" class="button bicolor button--icon">{% icon name="download" wrapped=1 %}{% trans 'Download XLSX' %}</a>
+                                    <div class="dropdown-toggle">{% icon name="arrow-down" %}</div>
+                                    <ul>
+                                        <li><a  class="button bicolor button--icon" href="?export=csv&{{ request.GET.urlencode }}">{% icon name="download" wrapped=1 %}{% trans 'Download CSV' %}</a></li>
+                                    </ul>
+                                </div>
```

Comment:
The same with the other template, this is the default header form the new modeladmin index, with the delete button added.